### DISCUSSION TITLE
How to leverage the jslt files in Swagger documentation

### DISCUSSION THREAD

**Discussion by lakshit78626** [schibsted/jslt#296] [Q&A]
Hi,

Any idea how to leverage the  jslt developed as part of transforming json request/response data, in swagger documentation for showing request/response structure just to avoid creating pojo classes.

Is it possible I am using apache camel for routing the data for which I need to make swagger documentation ? If yes then how kindly assist
Thanks
Lakshit

**Comment by larsga** [comment]
Not sure what you actually want here. You want to generate the Swagger/OpenAPI docs, I guess, but most of what's in there is outside of the scope of JSLT. Getting JSON examples should be easy: just run through some example inputs.
